pkg/accountdb/uow/uow.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uow
+
+import (
+	"context"
+	"storj.io/storj/pkg/accountdb/dbx"
+	"storj.io/storj/pkg/accountdb/repositories"
+)
+
+//TODO: add here all future created repositories, add unit tests
+// Used to manage db connections and context through different repositories
+type UnitOfWork interface {
```

Comment:
Not sure whether this is necessary.

Reply:
This is a good way to manage multiple repos and work with same instance of db conenction